lib/reek/source/source_locator.rb

```diff
@@ -11,60 +12,56 @@ class SourceLocator
       #
       # paths - a list of paths as Strings
       def initialize(paths)
-        @pathnames = paths.
-          map { |path| Pathname.new(path.chomp('/')) }.
-          flat_map { |pathname| current_directory?(pathname) ? pathname.entries : pathname }
+        @paths = paths.map(&method(:Pathname)).flat_map do |path|
```

Comment:
@mvz: Good point, sorry I missed that – GitHub didn’t refresh the comments after you wrote yours. I’ll fix along the other things you mentioned.